### PR TITLE
Improve meeting user relation handling

### LINKED ISSUE #2789: Client relation handling doesn't do back-relations anymore
It used to be that if one followed a relation field in a subscription, the contra-relation would automatically be filled for the other viewModel as well.

I.e. if one subscribed to a meeting and followed the motion_ids, the `ViewMeeting` would have a field `motions` with all of the corresponding `ViewMotion`s and the corresponding `ViewMotion`s would all have a field `meeting` that was filled with the `ViewMeeting` in question.

It seems like that has somehow stopped happening. 
There have recently been multiple bugs that were caused because a subscription followed a field, but the counter-relation wasn't named anywhere in the fieldset, causing the connection to either not be built, or only be built one-way (I'm not quite sure which).
An example of this would be #2710 which could not be fixed without adding the back-relation for the meeting-users `user_id` field into the fieldset of the follow.

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #2789 

Also cleans up all places were `meeting_user_ids` were subscribed.

### CODE REVIEW

**Comment by bastianjoel** [comment]
It might be that this comes with performance penalties in some scenarios. We should do some testing before merging this.

**Review by luisa-beerboom — APPROVED** [review]

**Comment by bastianjoel** [comment]
I tested this now a bit and could only observe a minor performance impact. 
To reduce this impact I added two minor tweaks that should especially improve autopilot performance. In my opinion this is ok to merge.

**Comment by bastianjoel** [comment]
I have a better Idea to make this more performant. Please do not merge yet.

**Review by luisa-beerboom — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

**Pull request merged**
